code/datums/ai/_ai_controller.dm

```diff
@@ -131,7 +134,7 @@ multiple modular subtrees with behaviors
 		return //this should remove them from processing in the future through event-based stuff.
 
 	if(!LAZYLEN(current_behaviors))
-		PerformIdleBehavior(delta_time) //Do some stupid shit while we have nothing to do
+		idle_behavior.perform_idle_behavior(delta_time, src) //Do some stupid shit while we have nothing to do
```

Comment:
This assumes that the idle behavior exists, which might not be true.

Reply:
it should.

Reply:
actually i guess not